src/infra/database/models/portal.js

```diff
@@ -0,0 +1,53 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const _portal = {};
+
+_portal.schema = new Schema({
```

Comment:
can you share a example data set that has been saved in the DB for this schema want to take a look at the data